docs/reference/execute-methods.md

```diff
@@ -199,11 +199,65 @@ bundleId | string | yes | The bundle identifier of the application to be removed
 
 #### Returned Result
 
-Either `true` if the app was successfully uninstalled, otherwise `false`
+Either `true` if the application was successfully uninstalled, otherwise `false`
 
 ### mobile: launchApp
 
-Executes the given app on the device under test. If the app is already running then it would be activated. If the app is not installed or cannot be launched then an exception is thrown.
+Executes the given application on the device under test. If the application is already running then it would be activated.
```

Comment:
If the application is already running then it would be restarted and activated.

Reply:
Calling launchApp against the running app didn't change the pid, so "restart" could be confusing

Reply:
It probably happens because of 

```
  if (!app.running) {
    app.launchArguments = arguments ?: @[];
    app.launchEnvironment = environment ?: @{};
    [app launch];
  } else {
    [app activate];
  }
```

Confusing, but yolo. Not sure if we want to break the legacy behavior